x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/helpers.ts

```diff
@@ -28,12 +24,7 @@ export const getQueryColumns = async (
   if ('esql' in performantQuery && performantQuery.esql) {
     performantQuery.esql = `${performantQuery.esql} | limit 0`;
   }
-  const table = await fetchDataFromAggregateQuery(
-    performantQuery,
-    dataView,
-    deps.data,
-    deps.expressions
-  );
+  const table = await fetchFieldsFromESQL(performantQuery, deps.expressions);
```

Comment:
ℹ️ This is irrelevant with this change but as I added the fetchFieldsFromESQL function which is lighter, I decided to also use it here.